Upstatement/puppy
# Update linting configs and bump node version to 14.16.0

## Changes

- Bump ESLint & Prettier config versions to 1.0.0 (and update peer dependencies)
- Update node version to 14.16.0 (the version of Puppeteer we're now using in puppy-lib requires 10.18.1+ so I figured I'd just bump to the most recent LTS version)

## Steps to test

- Check out the accompanying puppy-lib PR https://github.com/Upstatement/puppy-lib/pull/52
- In the puppy lib directory, run `npm link`
- In this repo run `npm link @upstatement/puppy`
- Then `npm install`
- `npm start` and make sure nothing is broken
- In your editor, make sure there's no ESLint errors

## Notes

- The diff is only so big because of the package-lock.json updates 😱
